api/src/main/java/com/capitalone/dashboard/service/GitRequestServiceImpl.java

```diff
@@ -1,18 +1,13 @@
 package com.capitalone.dashboard.service;
 
 import com.capitalone.dashboard.misc.HygieiaException;
-import com.capitalone.dashboard.model.Collector;
-import com.capitalone.dashboard.model.CollectorItem;
-import com.capitalone.dashboard.model.CollectorType;
-import com.capitalone.dashboard.model.Component;
-import com.capitalone.dashboard.model.DataResponse;
-import com.capitalone.dashboard.model.GitRequest;
-import com.capitalone.dashboard.model.QGitRequest;
+import com.capitalone.dashboard.model.*;
 import com.capitalone.dashboard.repository.CollectorRepository;
 import com.capitalone.dashboard.repository.ComponentRepository;
 import com.capitalone.dashboard.repository.GitRequestRepository;
 import com.capitalone.dashboard.request.GitRequestRequest;
 import com.mysema.query.BooleanBuilder;
+import com.vividsolutions.jts.util.CollectionUtil;
 import org.apache.commons.lang.StringUtils;
```

Comment:
Where is this CollectionUtils used? Use existing apache commons or spring's collection Util

Reply:
@tabladrum I have cleaned these up, It was my IDE auto handling the imports again, I'll will  configure my IDE settings again. 
I had to re-install it, and it went back to default settings.